### DISCUSSION TITLE
No person detected

### DISCUSSION THREAD

**Discussion by JeanCGR** [nextcloud/recognize#800] [Q&A]
Hi guys,

I need help getting facial recognition working in nextcloud

- I mounted a nextcloud image in a docker container in version 26.
- I installed the recognize app.
- I imported about ten photos with several people but after the scans (classify, cluster-faces), the person tab is still empty.
- But I have the people tag, portrait tag on the photos

I don't understand why the person tab is empty

PS: I have also tried using the contact app and adding profile pictures but that doesn't change anything.

Thanks in advance

**Comment by marcelklehr** [comment]
> I imported about ten photos

you need more than a hundred faces in your library before face clustering starts kicking in. Below that the clustering algorithm doesn't have enough data.

**Reply by JeanCGR** [reply]
Thank you very much, I understand better now
